### DISCUSSION TITLE
Suggestion: add donation/tip buttons to more places

### DISCUSSION THREAD

**Discussion by digitaldirk** [linkdotnet/Blog#357] [Ideas]
I think it would be nice to have a few more optional places to put the donation buttons (ko-fi, sponsor, patreon...). Under the introduction on the main page, left side of about me page. Alternatively, maybe a new link on the top right that's just for donations or a page footer? 

To selectively enable these how would you like this done - the config file or another mechanism?

Willing to do a PR but would like to discuss further.

**Comment by linkdotnet** [comment]
Oh absolutely - never thought of that. And thanks for the initiative 
For the selection: yes the config file with a new sub section might be the way to go. The whole section might be optional and if not set, should behave more or less the same as today.

in the end, it might be a collection of Boolean for all the specific places you have on mind. 

Does that somehow resonate with your idea?

**Reply by digitaldirk** [reply]
Yes thanks
